[2025-10-09T04:50:55.457+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: order_generator.load_orders scheduled__2025-10-05T06:00:00+00:00 [queued]>
[2025-10-09T04:50:55.474+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: order_generator.load_orders scheduled__2025-10-05T06:00:00+00:00 [queued]>
[2025-10-09T04:50:55.477+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2025-10-09T04:50:55.479+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 2
[2025-10-09T04:50:55.486+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2025-10-09T04:50:55.517+0000] {taskinstance.py:1304} INFO - Executing <Task(PythonOperator): load_orders> on 2025-10-05 06:00:00+00:00
[2025-10-09T04:50:55.531+0000] {standard_task_runner.py:55} INFO - Started process 2234 to run task
[2025-10-09T04:50:55.536+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'order_generator', 'load_orders', 'scheduled__2025-10-05T06:00:00+00:00', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/order_loader_dag.py', '--cfg-path', '/tmp/tmp38rtyzma']
[2025-10-09T04:50:55.549+0000] {standard_task_runner.py:83} INFO - Job 63: Subtask load_orders
[2025-10-09T04:50:55.720+0000] {task_command.py:389} INFO - Running <TaskInstance: order_generator.load_orders scheduled__2025-10-05T06:00:00+00:00 [running]> on host 81fbc710bbce
[2025-10-09T04:50:55.856+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=order_generator
AIRFLOW_CTX_TASK_ID=load_orders
AIRFLOW_CTX_EXECUTION_DATE=2025-10-05T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-10-05T06:00:00+00:00
[2025-10-09T04:50:55.942+0000] {logging_mixin.py:137} INFO - Successfully loaded data to orders
[2025-10-09T04:50:56.175+0000] {logging_mixin.py:137} INFO - Successfully loaded data to order_items
[2025-10-09T04:50:56.176+0000] {python.py:177} INFO - Done. Returned value was: None
[2025-10-09T04:50:56.190+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=order_generator, task_id=load_orders, execution_date=20251005T060000, start_date=20251009T045055, end_date=20251009T045056
[2025-10-09T04:50:56.246+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2025-10-09T04:50:56.267+0000] {taskinstance.py:2582} INFO - 0 downstream tasks scheduled from follow-on schedule check
